go.mod

```diff
@@ -4,39 +4,38 @@ go 1.20
 
 replace (
 	github.com/knative/pkg => github.com/rancher/pkg v0.0.0-20190514055449-b30ab9de040e
-	k8s.io/client-go => k8s.io/client-go v0.27.4
 	sigs.k8s.io/json => sigs.k8s.io/json v0.0.0-20211208200746-9f7c6b3444d2
 )
 
 require (
-	github.com/Masterminds/sprig/v3 v3.2.3
-	github.com/apparentlymart/go-cidr v1.1.0
-	github.com/aws/aws-sdk-go v1.44.272
+	github.com/Masterminds/sprig/v3 v3.2.2
+	github.com/apparentlymart/go-cidr v1.0.1
+	github.com/aws/aws-sdk-go v1.38.65
 	github.com/blang/semver v3.5.1+incompatible
-	github.com/coreos/go-semver v0.3.1
+	github.com/coreos/go-semver v0.3.0
 	github.com/docker/distribution v2.8.2+incompatible
-	github.com/docker/docker v24.0.2+incompatible
+	github.com/docker/docker v20.10.25+incompatible
 	github.com/docker/go-connections v0.4.0
 	github.com/ghodss/yaml v1.0.0
 	github.com/go-bindata/go-bindata v3.1.2+incompatible
-	github.com/go-ini/ini v1.67.0
-	github.com/mattn/go-colorable v0.1.13
-	github.com/mcuadros/go-version v0.0.0-20190830083331-035f6764e8d2
+	github.com/go-ini/ini v1.37.0
+	github.com/mattn/go-colorable v0.1.8
+	github.com/mcuadros/go-version v0.0.0-20180611085657-6d5863ca60fa
 	github.com/pkg/errors v0.9.1
-	github.com/rancher/norman v0.0.0-20230428185400-96b95e3475e0
-	github.com/sirupsen/logrus v1.9.2
-	github.com/stretchr/testify v1.8.2
-	github.com/urfave/cli v1.22.13
-	go.etcd.io/etcd/client/v2 v2.305.9
-	go.etcd.io/etcd/client/v3 v3.5.9
-	golang.org/x/crypto v0.11.0
-	golang.org/x/sync v0.2.0
-	google.golang.org/grpc v1.55.0
+	github.com/rancher/norman v0.0.0-20230831160711-5de27f66385d
+	github.com/sirupsen/logrus v1.9.3
+	github.com/stretchr/testify v1.8.4
+	github.com/urfave/cli v1.22.2
+	go.etcd.io/etcd/client/v2 v2.305.7
+	go.etcd.io/etcd/client/v3 v3.5.7
+	golang.org/x/crypto v0.12.0
+	golang.org/x/sync v0.3.0
+	google.golang.org/grpc v1.53.0
```

Comment:
While these dependencies look like they're downgraded due to docker change, the net diff for v1.5.0 from v1.4.x is same or updated: 
```
github.com/sirupsen/logrus v1.8.1 => v1.9.3 
github.com/stretchr/testify v1.8.0 => v1.8.4 
github.com/urfave/cli v1.22.2 => v1.22.2 
go.etcd.io/etcd/client/v2 v2.305.4 => v2.305.7
go.etcd.io/etcd/client/v3 v3.5.4 => v3.5.7 
google.golang.org/grpc v1.53.0 => v1.53.0 
```